### DISCUSSION TITLE
Docker Image got corrupted, getting error with Nextcloud AIO reinstallation

### DISCUSSION THREAD

**Discussion by NeZios** [nextcloud/all-in-one#4527] [Questions] [answered]
### Context
I am on UnraidOS and my docker image got corrupted. I deleted it, recreate it and reinstall all my containers. 
Everything is working fine. Except Nextcloud AIO giving me an error during the first start of **nextcloud-aio-nextcloud**.
I do not have backup and I have data on my Nextcloud. 
How should I proceed to make my Nextcloud work again without deleting my data ? 

Thanks in advance,

### Steps to reproduce
0. Requirement : having a fully working nextcloud AIO installation with data in it. 
1. Delete docker image and recreate it. 
2. Install nextcloud aio
3. Start nextcloud aio mastercontainer and download and install all the container. 

### Expected behavior 

nextcloud-aio-nextcloud container working normally

### Actual behavior 

Nextcloud container loop booting with this log 
<img width="963" alt="image" src="https://github.com/nextcloud/all-in-one/assets/38002539/33f5e31f-4901-4576-9192-08bf66516c5b">

### Host OS 

Unraid 6.12.4 

#### Nextcloud AIO version 

Nextcloud AIO 8.1.0

**Comment by szaimen** [comment, marked as answer]
Hi, so you deleted the mastercontainer and its volume and don't have a backup?

In that case, you need to follow https://github.com/nextcloud/all-in-one/blob/main/migration.md#migrate-only-the-files in order to recreate your instance.
